notebooks/data-sources/github/thoth_PR_EDA.ipynb

```diff
@@ -0,0 +1,1892 @@
+{
```

Comment:
<p>Same as above, should we also include repos which have no issues?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/490/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>